apps/core/scenes/passengerForm/BaggageBundles/BaggageTypeDetail.js

```diff
@@ -0,0 +1,118 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { StyleSheet, Icon } from '@kiwicom/universal-components';
+import type { IconNameType } from '@kiwicom/universal-components';
+import { Text } from '@kiwicom/margarita-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import { bags, type BagTitleType } from './BagTypes';
+
+type Props = {|
+  +type: BagTitleType,
+  +dimensions?: string,
+  +quantity: number,
```

Comment:
Why `dimensions` could be `undefined` but `quantity` no?

Reply:
Apparently forgot to change it after changing mocked data.Fixed it